stable/admin/values.yaml

```diff
@@ -82,6 +82,24 @@ admin:
   # Admin Service
   replicas: 1
 
+  # If specified and set to a value greater than 0, an initial membership
+  # consisting of server IDs <statefulset name>-0, <statefulset name>-1, ...,
+  # <statefulset name>-<n-1> is defined for all Admins. Defining this to a
+  # value larger than 1 allows a bootstrap server (e.g. <statefulset name>-0)
+  # that has to be reprovisioned from scratch to rejoin its peers when it is
+  # restarted, rather than bootstrapping a new domain consisting of itself.
+  #
+  # For multi-cluster configurations, only the Admin StatefulSet in the
+  # entrypoint cluster (i.e. the one with cloud.cluster.name equal to
+  # cloud.cluster.entrypointName) can specify bootstrapServers.
+  #
+  # For backwards-compatibility, the default value is 0 so that it is disabled
```

Comment:
random location. If this value is specified. We should not pass `--bootstrap-server-id` to nuodocker. See `nuoadmin` script.

Reply:
We do not pass `--bootstrap-server-id`. I was not able to find any place where the `NUODB_BOOTSTRAP_SERVERID` is defined.

Reply:
I see that there is a special `nuoadmin` script (it looks like the `nuoadmin` script in the Docker image with some extra stuff), which specifies `--bootstrap-server-id "$NUODB_BOOTSTRAP_SERVERID"`, but this will expand to `--bootstrap-server-id ""` (empty string) since it is not defined.

Reply:
The command-line argument expands to `--bootstrap-server-id ""`, because `NUODB_BOOTSTRAP_SERVERID` is never defined.

Reply:
Good catch. We should remove it then. Wanna do it as part of this review, or should I make a separate one?

Reply:
I don't think we need to change `nuoadmin`. It should accept `NUODB_BOOTSTRAP_SERVERID`. `bootstrapServers=<n>` is just a generalization of `NUODB_BOOTSTRAP_SERVERID` that assumes K8s conventions, and both should be available.

Reply:
I am not talking about `./build/docker/entry/nuoadmin`. I am talking about the overridden file that is specific to K8s.

Reply:
Do we really want the two to diverge further? I just looked, and the only difference is the handling of core files, which I'm not sure is even relevant:
```
$ diff -u build/docker/entry/nuoadmin ~/go/src/github.com/nuodb/nuodb-helm-charts/stable/admin/files/nuoadmin --ignore-space
--- build/docker/entry/nuoadmin	2020-01-08 18:00:25.000000000 -0500
+++ /Users/asuarez/go/src/github.com/nuodb/nuodb-helm-charts/stable/admin/files/nuoadmin	2020-03-01 22:27:30.000000000 -0500
@@ -24,6 +24,16 @@
 
 export NUODB_LOGDIR NUODB_CFGDIR NUODB_VARDIR NUODB_BINDIR NUODB_RUNDIR
 
+# attempt to retain the previous crash directory (within the configured window to avoid filling the disk)
+if [ -f $NUODB_CRASHDIR/* ]; then
+  crashcount=$(find $NUODB_LOGDIR/crash-* -type d -cmin -$OVERWRITE_WINDOW ! -path . | wc -l)
+  if [ $crashcount -lt $OVERWRITE_COPIES ]; then
+    crashbackupdir="$NUODB_LOGDIR/crash-$( date +%Y%m%dT%H%M%S )/"
+    mkdir $crashbackupdir
+    mv $NUODB_CRASHDIR/* $crashbackupdir
+  fi
+fi
+
 exec nuodocker start admin \
      --server-id "$NUODB_SERVERID" --alt-address "$NUODB_ALT_ADDRESS" --admin-port "$NUODB_ADMIN_PORT" \
      --domain-entrypoint "$NUODB_DOMAIN_ENTRYPOINT" --bootstrap-server-id "$NUODB_BOOTSTRAP_SERVERID" \
```

Reply:
good point. You are right.